### PR TITLE
Allow to autocomplete namespaced topics

### PR DESCRIPTION
**Pull request by ahcorde**
The autocomplete doesn't work with `namespaced` topics. Previous to this PR

https://github.com/ros-visualization/rqt/assets/1933907/99e1cbda-fc45-4698-8b33-4e1c8f743f38

With this PR:

https://github.com/ros-visualization/rqt/assets/1933907/9b5add45-2659-4150-bca7-1f98c44757f9

You can test this with `rqt_plot` or `rqt_gauges`

### CODE REVIEW

**Comment by ahcorde** [comment]
FYI @caguero

**Comment by ahcorde** [comment]
friendly ping @quarkytale

File: rqt_py_common/src/rqt_py_common/topic_completer.py [hunk 1]
```diff
@@ -46,6 +46,24 @@ def __init__(self, parent=None):
...
         # we need to separate array subscriptions by an additional /
+
+        topic_name = path
+
+        if self.topic_list:
+            # Remove backslash at the end of the topic name
+            if (topic_name[-1] == '/'):
+                topic_name = topic_name[:-1]
+
+            for topic, type in self.topic_list:
```
Comment (caguero) [inline rqt_py_common/src/rqt_py_common/topic_completer.py]: Do you need the `type`?

**Review by caguero — COMMENTED** [review]
Minor comment but other than that, it works for me.

File: rqt_py_common/src/rqt_py_common/topic_completer.py (hunk 1 above)
Comment (ahcorde) [reply rqt_py_common/src/rqt_py_common/topic_completer.py]: https://github.com/ros-visualization/rqt/pull/299/commits/26b0fe84d35368e035b2a004b8628d8c3824ba54

**Review by caguero — APPROVED** [review]

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/20292/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/14703/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/21022/)

**Pull request merged**